lib/cleanup/destroyable_records.rb

```diff
@@ -29,7 +29,8 @@ def print_data
 
     stdout.puts '********'
     stdout.puts "This provider has #{in_person_enrollments.size} in person enrollments " \
-                       "that will be destroyed"
+                   "that will be destroyed - Please handle these removals manually. " \
+                   "For more details check https://cm-jira.usa.gov/browse/LG-10679"
```

Comment:
if this is the case, should we be deactivating/soft deleting these SPs instead of destroying them?